src/generic/stepper/Header.jsx

```diff
@@ -29,16 +28,23 @@ export default function Header({ steps, className }) {
     >
       {steps.map(({ iconLabel, label, incomplete }, index) => {
         const isNotLastStep = index < steps.length - 1;
+        const isActiveStep = (isFirstStep && index === 0) || (!isFirstStep && index === 1);
         return (
         // eslint-disable-next-line react/no-array-index-key
           <React.Fragment key={`${index}-${label}`}>
-            <Icon className={incomplete ? 'bg-light-900' : 'bg-primary'}>
+            <Icon className={incomplete ? 'bg-light-900' : 'bg-primary'} size="1.5rem">
```

Comment:
you can use the `classnames` lib here to conditionally add classes.

Reply:
done

Reply:
@mehaknasir Is it done?